client/src/app/_services/account.service.ts

```
@@ -0,0 +1,61 @@
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AccountService {
+
+  private http = inject(HttpClient);
+  private baseUrl = environment.apiUrl;
+  currentUser = signal<User | null>(null);
+  roles = computed(() => {
+      const user = this.currentUser();
+      if (user && user.token) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)